### DISCUSSION TITLE
Images from developer accounts?

### DISCUSSION THREAD

**Discussion by ktsakalozos** [prometheus-operator/kube-prometheus#642] [General] [answered]
Hi,

Just noticed that there are a couple of images served from developer accounts (eg [1, 2]) and not from `quay.io/coreos` or `quay.io/prometheus`. Is this something we should expect to see more in the future or is it an oversight?

Thanks

[1] https://github.com/prometheus-operator/kube-prometheus/blob/release-0.6/manifests/prometheus-adapter-deployment.yaml#L28
[2] https://github.com/prometheus-operator/kube-prometheus/blob/release-0.6/manifests/setup/prometheus-operator-deployment.yaml#L27

**Comment by paulfantom** [comment, marked as answer]
That's where those projects live so that's why we point there. Cloning images to other organization doesn't solve anything.

However, if you need to have some sort of security compliance I recommend overriding all images (including ones from organizations) and not basing your deployment on upstream images.
